### ISSUE TITLE
dory$ git submodule update --init --recursive

### ISSUE THREAD

**Issue by Nagatejapallapu** [pulp-platform/dory#1]
uie24011@ozd1161u:~/Desktop/SDK/pulp-sdk/dory$ git submodule update --init --recursive
Cloning into '/home/uie24011/Desktop/SDK/pulp-sdk/dory/pulp-nn-1d'...
Username for 'https://github.com': nagatejapallapu
Password for 'https://nagatejapallapu@github.com': 
remote: Support for password authentication was removed on August 13, 2021. Please use a personal access token instead.
remote: Please see https://github.blog/2020-12-15-token-authentication-requirements-for-git-operations/ for more information.
fatal: Authentication failed for 'https://github.com/pulp-platform/pulp-nn-1d/'
fatal: clone of 'https://github.com/pulp-platform/pulp-nn-1d' into submodule path '/home/uie24011/Desktop/SDK/pulp-sdk/dory/pulp-nn-1d' failed
Failed to clone 'pulp-nn-1d'. Retry scheduled

**Comment by ESEQU** [comment]
I created a personal access token to solve this problem but then discovered that the links to the pulp-nn-1d repository and the pulp-nn-mixed repository don't work! I got the error message as follows:
`remote: Repository not found.
fatal: repository 'https://github.com/pulp-platform/pulp-nn-1d/' not found
fatal: clone of 'https://github.com/pulp-platform/pulp-nn-1d' into submodule path '/home/quentin/dory/pulp-nn-1d' failed
Failed to clone 'pulp-nn-1d'. Retry scheduled`
If you click on the links in the dory repository it also leads nowhere.
